pkg/scheduler/api/types.go

```diff
@@ -168,3 +168,9 @@ type ReservedNodesFn func()
 
 // VictimTasksFn is the func declaration used to select victim tasks
 type VictimTasksFn func() []*TaskInfo
+
+// QueueOverusedFn is the func declaration used to judge whether queue is overused.If overused, whether it is all dimensions overused.
+type QueueOverusedFn func(*QueueInfo) (bool, bool)
```

Comment:
We should not return two bool for an API.

Reply:
Because there exists two different scenario:

Reply:
find another way for that case; it's hard for user to understand the differentiation between those two return value.

Reply:
Another way is to separate it into two functions with different targets, but there exists similar function logic.